services/jenkins-pipeline-monitor/handler.py

```diff
@@ -123,8 +141,6 @@ def jenkins_pipeline_monitor():
     latest_day_builds = get_latest_day_builds(job, latest_build_number)
     logging.info(f'latest builds {latest_day_builds}')
     # filter latest day builds by desired build type a.k.a release job type
-    desired_release_job_type = ['mxnet_lib/static', 'python/pypi']
-
     filtered_builds = filter_by_desired_release_job_type(latest_day_builds, desired_release_job_type)
```

Comment:
we don't need to pass `desired_release_job_type` as an argument now that it is global

Reply:
Addressed